### DISCUSSION TITLE
Nightly release on iOS

### DISCUSSION THREAD

**Discussion by AHA1GE** [rustdesk/rustdesk#4665] [General]
Would you please consider publishing the beta version on TestFlight for iOS users?

**Comment by Hydro8** [comment]
Hi
I second this idea as we can’t connect from iOS to RustDesk if we use 1.2.4 and 2FA.

**Reply by xlionjuan** [reply]
Because iOS version is still 1.2.2

**Reply by Hydro8** [reply]
Yes so nightly on iOS will be top :)

**Comment by tschettervictor** [comment]
Good idea…

**Reply by tschettervictor** [reply]
Bump. Forgive me but it’s fairly important for me…
